src/hardware/vga_other.cpp

```diff
@@ -1476,23 +1477,29 @@ static void composite_settings(Section_prop &secprop)
 
 	auto int_prop = secprop.Add_int("hue", when_idle, hue.get_default());
 	int_prop->SetMinMax(hue.get_min(), hue.get_max());
-	int_prop->Set_help("Appearance of RGB palette. For example, adjust until the sky is blue.");
+	int_prop->Set_help(format_string("Appearance of RGB palette (%d by default).\n"
+	                                 "For example, adjust until the sky is blue.",
+	                                 hue.get_default()));
```

Comment:
I hope the defaults won't change too often, or it will be a nightmare for translators.

Reply:
They don't. But if they do, that's life 😄